### ISSUE TITLE
Set-Format fails when setting the last column with a NumberFormat

### ISSUE THREAD

**Issue by DarkLite1** [dfinke/ImportExcel#320]
Consider the following code:

```````powershell
$Data = @(
    [PSCustomObject]@{
        Nr1 = 5368709120
        Nr2 = 6166915072
    }
    [PSCustomObject]@{
        Nr1 = 10737418240
        Nr2 = 5368709120
    }
)

$ExportParams = @{
    Path          = Join-Path $env:TEMP 'test.xlsx'
    AutoSize      = $true
    FreezeTopRow  = $true
}

$Data | Export-Excel @ExportParams -CellStyleSB {
    Param (
        $WorkSheet,
        $TotalRows,
        $LastColumn
    )
    
    $WorkSheet.Cells.Style.HorizontalAlignment = 'Center'
 
    $PctCell = $WorkSheet.Cells['1:1'] | where Text -EQ 'Nr1'
    $WorkSheet.Column($PctCell.Start.Column) | Set-Format -NumberFormat '?\K\B'

    $PctCell = $WorkSheet.Cells['1:1'] | where Text -EQ 'Nr2'
    $WorkSheet.Column($PctCell.Start.Column) | Set-Format -NumberFormat '?\K\B'
    # Error: "ColumnMax can not span over existing column 4
    # Setting the last column with a NumberFormat fails
}
```````

The column of `Nr2` can not be formatted as an error is thrown.

**Comment by jhoneill** [comment]
This isn't a problem with the Set-Format command.  
If you take the | set-format out and do $x = $worksheet.column( .... )   you get the same result. 
This seems to be a bug in the underlying object, and I found that if I removed the call to set the horizontal alignment it started working. If I put the horizontal alignment in at the end, it still worked, so as a work-round I suggest moving that to the end.

However - if Horizontal alignment is set, and the file re-opened and one attempts to auto-size columns a similar error to the one originally reported.  (Use -ClearSheet to work round this) 

@dfinke do we have a channel to talk to the DLL author about this ? 

Cheers
James
